src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -1648,7 +1648,8 @@ describe("RedisSessionStorage#getLollipopAssertionRefForUser", () => {
       expect.any(Function)
     );
     expect(E.isRight(response)).toBeTruthy();
-    if (E.isRight(response)) expect(response.right).toEqual(anAssertionRef);
+    if (E.isRight(response) && O.isSome(response.right))
```

Comment:
Can we add a test case with `O.isNone(response.right)` ?

Reply:
Fixed in 0cadcce